cli/lsp/language_server.rs

```diff
@@ -121,6 +122,8 @@ pub struct Inner {
   maybe_import_map_uri: Option<Url>,
   /// An optional configuration for linter which has been taken from specified config file.
   pub maybe_lint_config: Option<LintConfig>,
+  /// An optional configuration for the test subcommand which has been taken from specified config file.
+  pub maybe_test_config: Option<TestConfig>,
```

Comment:
Is this used anywhere? I don't think we don't need to bother with this for now and could just revert this file?

Reply:
Done.